webview/src/setup/components/dvc/CliUnavailable.tsx

```diff
@@ -10,6 +10,30 @@ import {
   updatePythonEnvironment
 } from '../../util/messages'
 import { Warning } from '../../../shared/components/icons'
+import { ExtensionLink } from '../shared/ExtensionLink'
+import Tooltip from '../../../shared/components/tooltip/Tooltip'
+
+const PythonExtensionTooltip: React.FC<{
+  dataTestId: string
+  disabled: boolean
+  children: ReactElement
+}> = ({ dataTestId, disabled, children }) => (
+  <Tooltip
+    content={
+      <span data-testid={dataTestId}>
```

Comment:
Not sure if we need the extra tooltips since it does tell you to install the python extension in the text 🤔

Reply:
I don't think we need the tooltip either. It looks very strange with almost the same text popping up next to the original.